pkg/stanza/operator/input/windows/input.go

```diff
@@ -23,22 +23,23 @@ import (
 // Input is an operator that creates entries using the windows event log api.
 type Input struct {
 	helper.InputOperator
-	bookmark            Bookmark
-	buffer              Buffer
-	channel             string
-	maxReads            int
-	startAt             string
-	raw                 bool
-	excludeProviders    []string
-	pollInterval        time.Duration
-	persister           operator.Persister
-	publisherCache      publisherCache
-	cancel              context.CancelFunc
-	wg                  sync.WaitGroup
-	subscription        Subscription
-	remote              RemoteConfig
-	remoteSessionHandle windows.Handle
-	startRemoteSession  func() error
+	bookmark             Bookmark
+	buffer               Buffer
+	channel              string
+	maxReads             int
+	startAt              string
+	raw                  bool
+	supressRenderingInfo bool
```

Comment:
Just noticed the typo
```suggestion
	suppressRenderingInfo bool
```